scc/delivery-with-flux.hbs.md

```diff
@@ -8,7 +8,7 @@ To use Gitops Delivery with FluxCD, you must complete the following prerequisite
 
 - You must create a `Workload` that uses either the `source-to-url-package` or `basic-image-to-url-package` Carvel Package Supply Chain. See the [Carvel documentation](./carvel-package-supply-chain.hbs.md). You must have at least one Carvel `Package` generated by this `Workload` stored in your GitOps repository.
 - You must have at least one run cluster. run clusters serve as your deployment environments. They can either be Tanzu Application Platform clusters, or regular Kubernetes clusters, but they must have kapp-controller and Contour installed. See the [Carvel documentation](https://carvel.dev/kapp-controller/) and the [Contour documentation](https://projectcontour.io/).
-- You must create a Build cluster that has network access to your run clusters. It must have FluxCD Kustomize Controller installed. See the [Flux documentation](https://fluxcd.io/flux/installation/#dev-install).
```

Comment:
You need a cluster that has the FluxCD Kustomize Controller installed that has network access to your run clusters. You can reuse your build cluster for this if you want to.